### ISSUE TITLE
fuzzy completion result

### ISSUE THREAD

**Issue by linwaytin** [non-Jedi/eglot-jl#55]
When `eglot` is enable, the `completion-at-point` gives fuzzy results like the image shown here (https://github.com/julia-vscode/LanguageServer.jl/pull/906)

The fuzzy completion is good for function, but not for julia's latex input. For example, when I type `\eps` I would expect `\epsilon` is the first or second item in the completion list, but now it is deep in the long list and it's hard to get it. 

Is it possible to turn off the fuzzy completion in the language server? Or another possibility, is it possible to make the priority of julia's latex completion higher? I notice that in the variable `completion-at-point-functions`, the `eglot-completion-at-point-function` is the first one in the list, and then the two julia-mode-latexsub-completion-.... functions. Maybe moving `eglot-completion-at-point-function` after the latexsub functions can help?

Thank you!

**Comment by linwaytin** [comment]
I just tried the second approach, moving the `eglot-completion-at-point` after the functions provided by `julia-mode`. It works as expected. The latex completion starting with `\` is be done by the functions provided `julia`, unless it cannot find one. The rest will be provided by `eglot`.

**Comment by non-Jedi** [comment]
I'll need to think about whether there's a good, safe way of effecting the order that `eglot-completion-at-point` is inserted. I definitely agree that we want the latex ones from julia mode higher in the list. Pull request is welcome if you've made the change within eglot-jl and nothing is breaking catastrophically. Thanks!

**Comment by linwaytin** [comment]
Yes this is a subtle issue...actually I think it is the best is the Language server can handle this well by itself, or making the latex input using other way than `completion-at-point`, but both are not easy to do.

**Comment by non-Jedi** [comment]
> Yes this is a subtle issue...actually I think it is the best is the Language server can handle this well by itself, or making the latex input using other way than `completion-at-point`, but both are not easy to do.

I don't think I agree. Emacs already has a system for prioritizing completions. Working within that system seems like the right solution. If you want to switch over to latex substitutions not coming through `completion-at-point`, the latest version of `julia-mode` includes a function called `julia-latexsub-or-indent` that you can use in your config.

LanguageServer.jl actually does provide latex substitution completions as far as I know. The problem for you is just that they're not first in the list of completion candidates. Not sure if this is the LanguageServer not sorting its completion or eglot losing the sorting.

**Comment by linwaytin** [comment]
Yes, I mean LanguageServer (or eglot) should deal with this as part of the completion. 

By not using `completion-at-point`, I mean using like Emacs's input method. Actually, I cannot type latex in the minibuffer using the current `completion-at-point` approach. I use TeX-input-method as a workaround.
